topics/metabolomics/tutorials/qcxms-predictions/tutorial.md

```diff
@@ -0,0 +1,594 @@
+- To generate conformers and optimise them using semi-empirical methods.
+- To produce simulated mass spectra for a given molecule in MSP format.  
+
+time_estimation: 1H
+
+key_points:
+- Galaxy provides access to HPC resources and hence allows the use of semi-empirical methods and *in silico* prediction of QC-based mass spectra.
+- The shown workflow allows to simulate mass spectra of a molecule starting from its SMILES. 
+- xTB package is based on a semiempirical quantum mechanical method to optimise the generated conformers. 
+- QCxMS tool suite performs quantum chemistry calculations to simulate mass spectra. 
+
+contributions:
+  authorship:
+    - wee-snufkin
+    - hechth
+
+requirements :
+  - type: "internal"
+    topic_name: metabolomics
+    tutorials: 
+      - lcms-preprocessing
+      - gc_ms_with_xcms
+
+---
+
+Mass spectrometry (MS) is a powerful analytical technique used in many fields, including proteomics, metabolomics, drug discovery and many more areas relying on compounds identification. Even though nowadays MS is a standard and popular method, there are still compounds which lack experimental spectra. In those cases, predicting mass spectra from the chemical structure can reveal useful structural information, help in compound identification and expand the spectral databases, improving the accuracy and efficiency of database search. [{% cite Zhu2023 %}, {% cite Allen2016Computational %}]. 
+There have been several methods developed to predict mass spectra, which can be classified as either first-principles physical-based simulation or data-driven statistical methods [{% cite Zhu2023 %}]. To the first category we can assign purely statistical theories (quasi-equilibrium theory (QET) or Rice–Ramsperger–Kassel–Marcus (RRKM) theories) [{% cite Vetter1994 %}], as well as QCEIMS [{% cite Wang2020 %}] and semiempirical GFNn-xTB [{% cite Koopman2019 %}] which use Born–Oppenheimer molecular dynamics (MD) combined with fragmentation pathways. Data-driven statistical methods reach back to 1960s when DENDRAL project was started by early AI scientists [{% cite Lindsay1980ApplicationsOA %}] – it applied rule-based heuristic programming. More recently, CFM-ID has been introduced [{% cite Allen2014 %}, {% cite Allen2014metabolomics %}, {% cite Allen2016Computational %}, {% cite DjoumbouFeunang2019 %}, {% cite Wang2021 %}], which uses rule-based fragmentation and employs machine learning methods. Current advancements in machine learning led to recent work using deep neural networks that allow predicting spectra from molecular graphs or fingerprints [{% cite Wei2019 %}].
+
+You will be able to check out how QCxMS works in practice since we are going to use Galaxy tool suite based on this method [{% cite Grimme2013 %}, {% cite Bauer2014 %}, {% cite Bauer2016 %}]. Beforehand, we will generate conformers of the query molecule with [RDKit](http://www.rdkit.org) and we will use xTB for molecular optimisation [{% cite Bannwarth2020 %}]. 
```

Comment:
To which category belongs QCxms, I cant find it in the paragraph before

Reply:
it would be nice to explain why conformers and molecular optimisation are needed